core/vm/instructions.go

```diff
@@ -667,9 +663,6 @@ func opCreate2(pc *uint64, interpreter *EVMInterpreter, callContext *callCtx) ([
 	callContext.stack.Push(&stackValue)
 	callContext.contract.Gas += returnGas
 
-	if suberr == ErrExecutionReverted {
```

Comment:
So, here the assumption is made the when `interpreter.evm.Create2` returns `ErrExecutionReverted` in the last return value, the `res` is always `nil`. Need to check that.

Reply:
I think this is another case of the result not being used in the caller.